Lib/test/test_uuid.py

```diff
@@ -657,6 +658,11 @@ def testIssue8621(self):
 
             self.assertNotEqual(parent_value, child_value)
 
+    # bpo-35701: check that weak referencing to a UUID object can be created
```

Comment:
nitpick: usually we put the comment inside the function, not outside ;-)

Reply:
no worries, I'll move it inside. Usually for something like this I would fixup the commit and force push, is that ok to do here or is the convention to use separate commits for the entire process?

Reply:
I prefer to have multiple commits, they will be squashed into a single commit as the end anyway.

Reply:
Different commits are simpler to review.